roles/satellite-clone/tasks/pre_install_check.yml

```diff
@@ -17,48 +17,3 @@
 - name: Check if root has enough space
   fail: msg="Your root partition should atleast have {{ required_root_free_space }}GB but it contains only {{ root_free_space }}GB"
   when: root_free_space | int <= required_root_free_space | int
-
-- name: Check the presence of config tar file
```

Comment:
this task is applicable for standard backup and migration.  This PR refactors this task so as to account for standard and online backup - but misses considering the migration part.